Description: fix compile error in insertionops.cc regarding rename of s_frags_per_group
  Fixes:
    insertionops.cc: In function 'std::ostream& operator<<(std::ostream&, const ext2_super_block*)':
    insertionops.cc:66:47: error: 'const struct ext2_super_block' has no member named 's_frags_per_group'
  See git commit 412376efff3c0e0c2fea00666c2457e6f2ae1878
  "Add basic BIGALLOC support for cluster-based allocation" in
  upstreams e2fsprogs repository
Forwarded: yes
Author: Michael Prokop <mika@debian.org>
Last-Update: 2011-01-11

--- a/src/insertionops.cc
+++ b/src/insertionops.cc
@@ -63,7 +63,7 @@ std::ostream& operator<<(std::ostream& os, const ext2_super_block* const s_block
   os << "Block size: " << EXT2_BLOCK_SIZE(s_block) << std::endl;
   os << "Fragment size: " << EXT2_FRAG_SIZE(s_block) << std::endl;
   os << "# Blocks per group: " << s_block->s_blocks_per_group << std::endl;
-  os << "# Fragments per group: " << s_block->s_frags_per_group << std::endl;
+  os << "# Fragments per group: " << s_block->s_clusters_per_group << std::endl;
   os << "# Inodes per group: " << s_block->s_inodes_per_group << std::endl;
   os << "Mount time: " << s_block->s_mtime << std::endl;
   os << "Write time: " << s_block->s_wtime << std::endl;
